### PR TITLE
SDK-6311 access to frame data

### PR DESCRIPTION
**Pull request by TomekJurkowski**

### CODE REVIEW

File: components/BarcodePicker.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import {
-	requireNativeComponent,
```
Comment (TomekJurkowski) [inline components/BarcodePicker.js]: I didn't really make any changes in this file, apart from replacing tab-indentations with spaces (it was a mix of both before and it made reading the code painful)

Comment (lucatorella) [reply components/BarcodePicker.js]: thanks

**Comment by petradonka** [comment]
I looked at the PR, but without Kotlin/Android knowledge, I can't say much more than that "it looks good for me" 😄

**Comment by lucatorella** [comment]
@robertnawrot can you review this PR please?

**Pull request merged**